make/photon/prepare/models.py

```diff
@@ -0,0 +1,135 @@
+import os
+import logging
+from pathlib import Path
+from shutil import copytree, rmtree
+
+from g import internal_tls_dir, DEFAULT_GID, DEFAULT_UID, PG_GID, PG_UID
+from utils.misc import check_permission, owner_can_read, get_realpath
+
+
+class InternalTLS:
+
+    harbor_certs_filename = {
+        'harbor_internal_ca.crt',
+        'proxy.crt', 'proxy.key',
+        'core.crt', 'core.key',
+        'job_service.crt', 'job_service.key',
+        'registryctl.crt', 'registryctl.key',
+        'registry.crt', 'registry.key'
+    }
+
+    clair_certs_filename = {
+        'clair_adapter.crt', 'clair_adapter.key',
+        'clair.crt', 'clair.key'
+    }
+
+    notary_certs_filename = {
+        'notary_signer.crt', 'notary_signer.key',
+        'notary_server.crt', 'notary_server.key'
+    }
+
+    chart_museum_filename = {
```

Comment:
Because we are not going to enable TLS in chartmuseum, should remove it?

Reply:
TLS still supported in the chart. just not enable mutual TLS in it.